test/metabase/util/encryption_test.clj

```diff
@@ -118,3 +124,43 @@
   (testing "Something that is not encrypted, but might be should return the original text"
     (is (= fake-ciphertext
            (encryption/maybe-decrypt secret fake-ciphertext)))))
+
+(deftest ^:parallel stream-encryption-test
+  (testing "Can encrypt stream"
+    (let [input-stream (ByteArrayInputStream. (.getBytes "test string"))
+          encrypted-stream (encryption/encrypt-stream secret input-stream)
+          output-string (slurp encrypted-stream)]
+      (is (not= "test string" output-string))))
+  (testing "Can encrypt and decrypt streams"
```

Comment:
nit - your call, but I personally find repeated `testing` forms to be more readable with newlines between them 
```suggestion

  (testing "Can encrypt and decrypt streams"
```